loopchain/peer/block_manager.py

```diff
@@ -545,24 +618,21 @@ def __block_request_to_peers_in_sync(self, peer_stubs, my_height, unconfirmed_bl
         :param max_height:
         :return: my_height, max_height
         """
+
         peer_index = 0
 
         while max_height > my_height:
             if self.__channel_service.state_machine.state != 'BlockSync':
                 break
+            
+            await asyncio.sleep(0.1)
```

Comment:
How about using a configuration instead of the constant(`0.1`)?

Reply:
The 0.1 value moved to variable which located configure file.
 - variable name : ASYNC_TINY_SLEEP_TIME